appveyor.yml

```diff
@@ -10,14 +10,14 @@ environment:
   app_id: inputstream.rtmp
 
   matrix:
-    - GENERATOR: "Visual Studio 14"
+    - GENERATOR: "Visual Studio 15"
       CONFIG: Release
-    - GENERATOR: "Visual Studio 14 Win64"
+    - GENERATOR: "Visual Studio 15 Win64"
       CONFIG: Release
-    - GENERATOR: "Visual Studio 14 Win64"
+    - GENERATOR: "Visual Studio 15 Win64"
       CONFIG: Release
       WINSTORE: -DCMAKE_SYSTEM_NAME=WindowsStore -DCMAKE_SYSTEM_VERSION="10.0.16299.0"
-    - GENERATOR: "Visual Studio 14 ARM"
+    - GENERATOR: "Visual Studio 15 ARM"
```

Comment:
Same VS2015?

Reply:
@Rechi has changed also some other Leia addons to them, only he mean some time OK, the cmake version must stay at 10.0.16299.0

Reply:
Ok, cool